### PR TITLE
Refactor Motors_Tab

### PR DESCRIPTION
**Pull request by pkaig**
Updated Motor Tab

### CODE REVIEW

File: src/js/tabs/motors.js [hunk 1]
```diff
@@ -186,26 +187,42 @@ TABS.motors.initialize = function (callback) {
...
             tempBar.toggle(self.isEscSensorEnabled);
             errorBar.toggle(FC.MOTOR_CONFIG.use_dshot_telemetry);
 
             meterLabel(thrBar, '0%', '100%');
             meterLabel(rpmBar, 0, rpmMax);
+            meterLabel(headspeedBar, '0RPM', headspeedMax + 'RPM');
+            meterLabel(tailspeedBar, '0RPM', tailspeedMax + 'RPM');
             meterLabel(voltBar, '0V', voltMax.toFixed(0) + 'V');
             meterLabel(currBar, '0A', currMax.toFixed(0) + 'A');
             meterLabel(tempBar, '0&degC', '150&degC');
```
Comment (egonl) [inline src/js/tabs/motors.js:228]: Should be &deg;

File: src/js/tabs/motors.js [hunk 2]
```diff
@@ -186,26 +187,42 @@ TABS.motors.initialize = function (callback) {
...
+            headspeedBar.toggle((self.isFreqSensorEnabled || self.isEscSensorEnabled || FC.MOTOR_CONFIG.use_dshot_telemetry) && motorIndex == 0);
+            tailspeedBar.toggle((self.isFreqSensorEnabled || self.isEscSensorEnabled || FC.MOTOR_CONFIG.use_dshot_telemetry) && motorIndex == 1);
             voltBar.toggle(self.isEscSensorEnabled);
             currBar.toggle(self.isEscSensorEnabled);
             tempBar.toggle(self.isEscSensorEnabled);
             errorBar.toggle(FC.MOTOR_CONFIG.use_dshot_telemetry);
 
             meterLabel(thrBar, '0%', '100%');
             meterLabel(rpmBar, 0, rpmMax);
+            meterLabel(headspeedBar, '0RPM', headspeedMax + 'RPM');
```
Comment (egonl) [inline src/js/tabs/motors.js:224]: Maybe a space between value and RPM?

**Review by egonl — APPROVED** [review]
Looks good!

File: src/js/tabs/motors.js (hunk 1 above)
Comment (pkaig) [reply src/js/tabs/motors.js]: Added.

File: src/js/tabs/motors.js (hunk 2 above)
Comment (pkaig) [reply src/js/tabs/motors.js]: Done. Cheers Egon.

**Pull request merged**